examples/routing_service/routing_service_msql_adapter/cpp/rs_main/rsmain.cxx

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Perhaps this concept could be illustrated in a separate example. Having two targets and doubling the number of files does make this a little bit more complicated.

Also, having the main function in a subfolder seems sensible, but it's also hard to follow. You could have everything in the top folder and build both targets (the DLL and the EXE) in the same CMakeLists.txt file.

Reply:
I am only doubling the CMakeLists.txt and README.md files. The source is common for both except for rsmain.cpp.